parsers/corelight-sensor.yaml

```diff
@@ -44,42 +411,126 @@ script: |
-      Vendor.result = /failure/i | event.outcome := "failure";
-      Vendor.result = "encrypted" | event.outcome := "success";
-      Vendor.result = "SSL_NOT_ALLOWED_BY_SERVER" | event.outcome := "failure";
-      Vendor.result = "HYBRID_REQUIRED_BY_SERVER" | event.outcome := "failure";
-      Vendor.result = "SSL_CERT_NOT_ON_SERVER" | event.outcome := "failure";
-      Vendor.result = "SSL_WITH_USER_AUTH_REQUIRED_BY_SERVER" | event.outcome := "failure";
-      Vendor.result = "INCONSISTENT_FLAGS" | event.outcome := "failure";
-      Vendor.result = "SSL_REQUIRED_BY_SERVER" | event.outcome := "failure";
+      _path = "known*"
+          | array:append("event.category[]", values=["host"])
+          | array:append("event.type[]", values=["info"]);
+
+      in(_path, values=["conn*", "specific_dns_tunnels", "generic_dns_tunnels", "tunnel", "vpn", "stun*"])
+          | array:append("event.category[]", values=["network"])
+          | array:append("event.type[]", values=["connection"]);
+
+      in(_path, values=["corelight*", "etc_viz", "suricata_stats", "zeek_doctor", "mqtt*"])
+          | array:append("event.category[]", values=["process"])
+          | array:append("event.type[]", values=["info"]);
+
+      in(_path, values=["dce_rpc", "dns*", "dhcp", "encrypted_dns", "ftp", "irc", "ntp", "dga", "ocsp"])
+          | array:append("event.category[]", values=["network"])
+          | array:append("event.type[]", values=["connection", "protocol", "info"]);
+
+      in(_path, values=["dpd", "rfb", "traceroute", "ssdp"])
+          | array:append("event.category[]", values=["network"])
+          | array:append("event.type[]", values=["connection", "info"]);
+
+      in(_path, values=["files*", "pe"])
+          | array:append("event.category[]", values=["file"])
+          | array:append("event.type[]", values=["info"]);
+
+      _path = "http*"
+          | array:append("event.category[]", values=["network", "web"])
+          | array:append("event.type[]", values=["connection", "protocol", "info"]);
+
+      _path = "intel"
+          | event.kind := "alert"
+          | array:append("event.category[]", values=["threat", "network"])
+          | array:append("event.type[]", values=["indicator"])
+          | rule.name := "intel";
+
+      _path = "suricata_corelight"
+          | event.kind := "alert"
+          | array:append("event.category[]", values=["threat", "network"])
+          | array:append("event.type[]", values=["indicator"])
+          | rule.name := Vendor.alert.signature;
+
+      _path = "notice"
+          | event.kind := "alert"
+          | array:append("event.category[]", values=["threat", "network"])
+          | array:append("event.type[]", values=["indicator"])
+          | rule.name := Vendor.msg;
+
+      _path = "mysql"
+          | array:append("event.category[]", values=["network", "database"])
+          | array:append("event.type[]", values=["connection", "protocol"]);
```

Comment:
from CRWD
- can be here set `access `type corresponding to `database `category?